### PR TITLE
Fixes manual balance adding after editing one

### LINKED ISSUE #1298: Adding/editing manual balances does not work correctly
## Problem Definition

Adding the first manual balance works fine. After that we can't add any other one. Label seems to be uneditable. The add new balance menu feels like editing a balance.

Peek 2020-08-09 10-59

## Task

Fix it.
Also make sure it all works fine in the future by perhaps adding a test.

### PR DESCRIPTION
**Pull request by kelsos**
Closes #1298

### CODE REVIEW

File: docs/changelog.rst [hunk 1]
```diff
@@ -2,6 +2,7 @@
 Changelog
 =========
 
+* :bug' `1298` Fix an issue where it was not possible to add new manual balances after editing one.
```
Comment (LefterisJP) [inline docs/changelog.rst:5]: Was that an issue also in v1.6.1? I did not really check. If not then we don't really need a changelog. Plus note the changelog should be :bug: and not :bug'

But no worries I will fix that in the next PR. no need to push again for this.

Comment (LefterisJP) [reply docs/changelog.rst:5]: Okay I just checked. No need for a changelog entry. This issue is not present in 1.6.1 :)

Comment (isidorosp) [reply docs/changelog.rst:5]: Yes, this issue existed in 1.6.1.

Comment (LefterisJP) [reply docs/changelog.rst:5]: Can you rebase and remove the changelog entry @kelsos? If not I can do it for you.

Comment (LefterisJP) [reply docs/changelog.rst:5]: @isidorosp Really? It existed? I added a balance with a new account in v1.6.1 and then edited it and tried to add another manual balance and it worked.

Comment (isidorosp) [reply docs/changelog.rst:5]: To be honest I didn't test it but I can't see (based on the changes in the code) why this bug _wouldn't_ also exist in 1.6.1... Downloading packaged 1.6.1 now to test.

Comment (LefterisJP) [reply docs/changelog.rst:5]: Correction: It exists. Please just rebase on develop, fix typo and we merge.

**Pull request merged**